sdk/cosmos/azure-cosmos/README.md

```diff
@@ -750,6 +750,52 @@ not being able to recognize the new NonStreamingOrderBy capability that makes ve
 If this happens, you can set the `AZURE_COSMOS_DISABLE_NON_STREAMING_ORDER_BY` environment variable to `"True"` to opt out of this
 functionality and continue operating as usual.*
 
+### Public Preview - Full Text Policy and Full Text Indexes
+We have added new capabilities to utilize full text policies and full text indexing for users to leverage full text search
+utilizing our Cosmos SDK. These two container-level configurations have to be turned on at the account-level
+before you can use them.
+
+A full text policy allows the user to define the default language to be used for all full text paths, or to set
+a language for each path individually in case the user would like to use full text search on data containing different
+languages in different fields.
+
+A sample full text policy would look like this:
+```python
+full_text_policy = {
+    "defaultLanguage": "en-US",
+    "fullTextPaths": [
+        {
+            "path": "/text1",
+            "language": "en-US"
+        },
+        {
+            "path": "/text2",
+            "language": "en-US"
+        }
+    ]
+}
+```
+Currently, the only supported language is `en-US`.
```

Comment:
Supported is one question - what happens when another language is chosen? Error? Language ignored and handled like en-US? What are the plans to extend language support? Never? Long-term? short-term?

Reply:
this is a public document that does not exist yet afaik - I'll go ahead and update on the specifics I can do at the moment